src/SFA.DAS.ApplyService.InternalApi.UnitTests/Services/AssessorLookupServiceTests.cs

```diff
@@ -56,5 +56,16 @@ public void GetTitleForPage_when_invalid_page_returns_null(string invalidPageId)
 
             Assert.That(actualTitle, Is.Null);
         }
+
+        [Test]
+        public void GetSectorIdsForSectorPageId()
+        {
+            var pageId = RoatpWorkflowPageIds.DeliveringApprenticeshipTraining.AgricultureEnvironmentalAndAnimalCare.MostExperiencedEmployee;
+            var result = _assessorLookupService.GetSectorQuestionIdsForSectorPageId(pageId);
+
+            Assert.AreEqual(
```

Comment:
Can you use CollectionAssert.AreEquivalent here instead?

Reply:
They are not collections.  They're a class which contains a lot of fields that are Ids

Reply:
I tried to use the .That and 'AreSame'/'AreEqual' convention to compare, but Nunit wasn't having any of it

Reply:
Fair enough. Marking as resolved